src/modules/SpaceCalculation.rb

```diff
@@ -578,183 +559,147 @@ def get_partition_info
       # remove the previous failures
       @failed_mounts = []
 
-      # installation stage - Storage:: is definitely present
-      # call Storage::GetTargetMap()
-      targets = Convert.convert(
-        WFM.call("wrapper_storage", ["GetTargetMap"]),
-        :from => "any",
-        :to   => "map <string, map>"
-      )
+      target_partitions = []
+      min_spare = 20 * 1024 * 1024 # minimum free space ( 20 MB )
```

Comment:
This is old code. As said in the description of the pull request, some old code looks like new due to re-indentation. The scope of the PBI was to substitute the calls to the old libstorage with calls to the new one. I don't think rewriting/refactoring the old code should be done now.